tests/unit/utils/test_get_marketplace_tags.py

```diff
@@ -7,16 +7,28 @@
 
 class TestGetMarketplaceTags(unittest.TestCase):
 
+  MOCK_MARKETPLACE_PRODUCTT_CODE_SC = {
+    "Parameter": {
+      "Name": "/service-catalog/MarketplaceProductCodeSC",
+      "Type": "String",
+      "Value": "mkt-cust-1234",
+      "Version": 1,
+      "LastModifiedDate": "today",
+      "ARN": "arn:aws:ssm:us-east-1:1111111111:parameter/service-catalog/MarketplaceProductCodeSC",
+      "DataType": "text"
+    }
+  }
+
   def test_has_tags(self):
     ssm = boto3.client('ssm')
     with Stubber(ssm) as stubber, \
       patch('set_tags.utils.get_marketplace_customer_id') as customer_id_mock, \
       patch('set_tags.utils.get_ssm_parameter') as ssm_param_mock:
         customer_id_mock.return_value = "mkt-cust-1234"
-        ssm_param_mock.return_value = "mkt-prod-1234"
+        ssm_param_mock.return_value = self.MOCK_MARKETPLACE_PRODUCTT_CODE_SC
         result = utils.get_marketplace_tags(1234567)
         expected = [
-          {'Key': 'marketplace:productCode', 'Value': 'mkt-prod-1234'},
+          {'Key': 'marketplace:productCode', 'Value': 'mkt-cust-1234'},
```

Comment:
Not sure why you made this change.

Reply:
this is because I incorrectly set `"Value": "mkt-cust-1234"` in MOCK_MARKETPLACE_PRODUCT_CODE_SC which was wrong.  I have fixed that issue.